NoSQL/week1/README.md

```diff
@@ -1,23 +1,30 @@
 # Week 1: Setting Up
 
-This week will focus on three topics: learning how to use Git Bash and Github, creating databases, and accessing databses.
+This week will focus on three topics: learning how to use Git Bash and Github, creating databases, and accessing databases.
 
 ### 📚 Summary
-...
+Students are able to navigate MongoDB Compass and understand how GitHub/Git Bash can be used effectively for developing applications after this week. They will experiment with and create sample datasets that reinforce understanding of how NoSQL databases store information and are structured.
 
 ### Learning Objectives
```

Comment:
I reformatted your learning objectives in a more readable format below. I agree with the additions you made in comparison to the learning objectives I wrote for week 1, so I'm also implementing those in the root readme as well.

- How to Use GitHub (commits, cloning, branches etc.) and Git Bash
- MongoDB Compass: access and create your databases
- Basic MongoDB terminology
- The basics of JSON
- Installing an IDE that supports JS